client/src/App.js

```diff
@@ -1,25 +1,38 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter as Router, Switch } from 'react-router-dom';
+import GlobalStyles from './GlobalStyles';
+import Popular from './pages/Popular';
+import Recs from './pages/Recs';
+import Lists from './pages/Lists';
+import Authentication from './pages/Authentication';
+import BasicLayout from './layouts/BasicLayout';
+import HeaderBottomNavLayout from './layouts/HeaderBottomNavLayout';
+import HeaderSideBottomNavLayout from './layouts/HeaderSideBottomNavLayout';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Router>
+        <GlobalStyles />
+        <Switch>
+          <BasicLayout exact path="/login" component={Authentication} />
```

Comment:
`<BasicLayout>` and the other Layouts are your Routes. A layout shouldn't be a route :).
You can rename it to fix this issue (same for the other routes).

```suggestion
          <BasicLayoutRoute exact path="/login" component={Authentication} />
```